spec/models/piece_spec.rb

```diff
@@ -225,4 +225,25 @@
       end
     end
   end
+
+  describe "#is_move_malformed?" do
+    let(:game) {create :game}
```

Comment:
Creating a piece via the `:piece` factory will also lead to the creation of a game due to the association in the factory definition, so if you're going to be creating a piece at the start of every test, you can just do that via `let` instead: `let(:white_king) {create :piece}`.